src/ui/AP2DataPlot2D.h

```diff
@@ -257,7 +257,7 @@ private slots:
     qint64 m_startIndex; //epoch msecs since graphing started
     QAction *m_addGraphAction;
     UASInterface *m_uas;
-    QSharedPointer<QProgressDialog> m_progressDialog;
```

Comment:
http://doc.qt.io/qt-5/qsharedpointer.html

QSahred pointer is always constructed as NULL and it deletes what is referenced when it goes out of scope. So the valugrind messages are not valid in this case.